traffic_portal/app/src/common/modules/form/cdnDnssecKeys/generate/form.GenerateCdnDnssecKeys.tpl.html

```diff
@@ -56,8 +56,22 @@
             <div class="form-group" ng-class="{'has-error': hasError(cdnGenerateDnssecKeysForm.effectiveDate), 'has-feedback': hasError(cdnGenerateDnssecKeysForm.effectiveDate)}">
                 <label class="control-label col-md-2 col-sm-2 col-xs-12">Effective Date (UTC) *</label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input name="effectiveDate" type="text" class="form-control" ng-model="dnssecKeysRequest.effectiveDate" required autofocus>
+                    <input class="form-control"
+                           placeholder="Select a date..."
+                           moment-picker="dnssecKeysRequest.effectiveDate"
+                           format="YYYY-MM-DD HH:mm:ss"
+                           start-view="month"
+                           today="true"
+                           name="effectiveDate"
+                           type="text"
+                           ng-model="dnssecKeysRequest.effectiveDate"
+                           ng-model-options="{ updateOn: 'blur' }"
+                           ng-pattern="/^[0-9\:\-\ ]+$/"
+                           change="ctrl.zeroSeconds()"
+                           required
+                           autofocus>
                     <small class="input-error" ng-show="hasPropertyError(cdnGenerateDnssecKeysForm.effectiveDate, 'required')">Required</small>
+                    <small class="input-error" ng-show="hasPropertyError(cdnGenerateDnssecKeysForm.effectiveDate, 'pattern')">Invalid date format</small>
```

Comment:
can you add this to the message so it's clear what the expected format is? YYYY-MM-DD HH:mm:ss

Reply:
Validation message changed to show format